tests/test_tools.py

```diff
@@ -104,6 +105,55 @@ def test_get_block_details_live(tools_live):
     assert "createdAt" in details
 
 
+def test_get_credits_balance(tools_mock):
+    balance = tools_mock.get_credits_balance()
+    assert isinstance(balance, dict)
+    assert "balance" in balance
+    assert True
+
+
+@pytest.mark.live
+def test_get_credits_balance_live(tools_live):
+    balance = tools_live.get_credits_balance()
+    assert isinstance(balance, dict)
+    assert "balance" in balance
+
+
+def test_get_credits_history(tools_mock):
+    credits_history = tools_mock.get_credits_history()
+    assert isinstance(credits_history, dict)
+    assert "content" in credits_history
+    assert isinstance(credits_history["content"], list)
+    assert len(credits_history["content"]) == 1
+
+
+@pytest.mark.live
+def test_get_credits_history_live(tools_live):
+    balance_history = tools_live.get_credits_history(
+        start_date="2022-03-01", end_date="2022-03-02"
+    )
+    assert isinstance(balance_history, dict)
+    assert "content" in balance_history
+    assert len(balance_history["content"]) == 32
+    assert isinstance(balance_history["content"], list)
+
+
+@pytest.mark.live
+def test_get_credits_history_no_bounds_live(tools_live):
```

Comment:
What this is really checking is that the function code can handle a not defined start (or enddate), not so much the content of the response. Therefore could also just be a mocktest. Ideally could be using @pytest.parameterize .

Reply:
Done!